Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_3.h

```diff
@@ -179,6 +179,43 @@
  *              EXP2                                              EXP1
  */
 #if HAS_SPI_LCD
+
+  #if ENABLED(ANET_FULL_GRAPHICS_LCD)
+    /** Be careful! To use this display you have to do some hardware modifications to the wires and the connectors.
```

Comment:
What about turning this into   `#error "Be careful! To use this display you have to do some hardware modifications to the wires and the connectors. See motherboard pins file for details. Comment out this line to continue."` instead? Then it will make sure users read this warning.

Reply:
This is far safer to prevent the user from making a mistake.
So what do I have to do now to make this change possible? This is my first commit on github.

Reply:
You can make the change & then push a new commit, but I'm wondering if we need to be this cautious. @thinkyhead?

Reply:
> what do I have to do now to make this change possible?

Github should let you edit the file right from this here web page.